app/code/Magento/Checkout/view/frontend/web/template/minicart/item/default.html

```diff
@@ -45,7 +45,7 @@
                                 <span data-bind="html: option.value.join('<br>')"></span>
                             <!-- /ko -->
                             <!-- ko ifnot: Array.isArray(option.value) -->
-                                <span data-bind="text: option.value"></span>
+                                <span data-bind="html: option.value"></span>
```

Comment:
This changes is forbidden due to security reasons. Please contact security team about that